docs/docs/attestation/sbom.md

```diff
@@ -49,3 +50,30 @@ You can verify attestations.
 ```
 $ COSIGN_EXPERIMENTAL=1 cosign verify-attestation <IMAGE>
 ```
+
+## Scanning
+
+Trivy can take an SBOM attestation as input and scan for vulnerabilities. Currently, Trivy supports CycloneDX-type attestation.
+
+In the following example, Cosign can get an attestation and trivy scan it.
+
+```bash
+$ cosign verify-attestation --key /path/to/cosign.pub --type cyclonedx <IMAGE> > sbom.cdx.intoto.jsonl
```

Comment:
If it includes several attestations, Trivy fails to scan, right?

Reply:
The `--type cyclonedx` option of Cosign will take only cyclonedx attestation if the image has multiple attestations.
Depending on the number of cyclonedx attestations, the behavior will be as follows.
-  If there is no cycnlonedx attestation, the cosign command will fail.
- If there is one cycnlonedx attestation, Trivy will successfully scan. This situation is assumed in the text of the description.
- (rare case) If there is more than one cycnlonedx attestation, it will also successfully scan because json.NewReader parses only the first line if there are multiple lines in input data([ref](https://go.dev/play/p/c6Gf9_O4_G5)). This is unintended behavior but it is not bad because they may be the same content.

[jsonnew]: https://github.com/aquasecurity/trivy/blob/01123854b4db8a6c7368829e47197018b3a581c2/pkg/fanal/artifact/sbom/sbom.go#L121

Reply:
>If there is more than one cycnlonedx attestation, it will also successfully scan because json.NewReader parses only the first line if there are multiple lines in input data([ref](https://go.dev/play/p/c6Gf9_O4_G5)).

I didn't know that. Cool. Thanks for your explanation.